src/components/DeleteItemButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { doc, deleteDoc } from '@firebase/firestore';
+import { db } from '../lib/firebase';
+import Modal from './Modal';
+
+function DeleteItemButton(item) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleDelete = async ({ userToken, item, onClose }) => {
+    setIsOpen(false);
+    const docRef = doc(db, 'users', userToken, 'list', item);
+    await deleteDoc(docRef);
+  };
+
+  return (
+    <div>
+      <button onClick={() => setIsOpen(true)}>Delete</button>
+
+      <Modal open={isOpen}>
+        <p id="dialog-description">Are you sure you want to delete?</p>
```

Comment:
If you remove the `aria-describedBy` you won't need this `id` any longer

Reply:
Okay, I'm curious if this is considered redundant because the role is alert? 

When I read through this part of the [WAI-ARIA documentation](https://www.w3.org/TR/wai-aria-practices-1.2/#alertdialog), I thought it was suggesting to use the `aria-describedBy` property when role="alert" is used.

Reply:
Ah, I see your suggestion on changing this to `role=dialog`. I'll be sure to read up on that. Thank you!